programs/openbook-v2/fuzz/Cargo.toml

```diff
@@ -8,8 +8,8 @@ edition = "2021"
 cargo-fuzz = true
 
 [dependencies]
-anchor-lang = "0.28.0"
-anchor-spl = "0.28.0"
+anchor-lang = { git = "https://github.com/coral-xyz/anchor.git", rev = "2af9cc669", version = "0.28.0" }
```

Comment:
unfortunately this will break fuzz code. We need to use the same version of`anchor-lang` as in openbook,  but deriving `arbitrary` traits for the ix data, something like we did here: https://github.com/openbook-dex/anchor/tree/v0.28.0_abfdc4e-arbitrary

also `patch` entry below should be updated to not patch the default `crates-io` registry but the `anchor.git` repo

Reply:
just updated the anchor arbitrary patch:
https://github.com/openbook-dex/anchor/tree/v0.28.0_2af9cc66-arbitrary

Reply:
thx. I move the git-Urls to patched section. did you mean that?

see [patched anchor](https://github.com/openbook-dex/openbook-v2/pull/168/commits/509a65de7d4a710b8e85d05f4a4bcaf81ebfac0e)